scripts/workarounds/boot-order/metal-lib.sh

```diff
@@ -0,0 +1,510 @@
+    # "like-vendors" means their efibootmgr outboot matches
+
+    # formatting:
+    # if another vendor is identical then it should live with another.
+    # vendors may have differing hostnames, depending where this script runs
+    # vendor)
+    #   hostname_prefix_1)
+    #     file1)
+    #     fileN)
+    #   hostname_prefix_2)
+    #     file1)
+    #     fileN)
+    #   hostname_prefix_N)
+    #     file1)
+    #     fileN)
+    #   error)
+    #   remove_file_1
+    #   remove_file_N
+    # done
+    case $vendor in
+        *GIGABYTE*)
+            # Removal file(s) ...
+            efibootmgr | grep -ivP '(pxe ipv?4.*)' | grep -iP '(adapter|connection|nvme|sata)' | tee /tmp/rbbs1
+            efibootmgr | grep -iP '(pxe ipv?4.*)' | grep -i connection | tee /tmp/rbbs2
+            efibootmgr_prefix=''
+            efi_trim
+            efi_specials
+            efi_remove
+            case $hostname in
+                ncn-m*)
+                    efibootmgr | grep -iP '(pxe ipv?(4|6).*adapter)' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    efibootmgr | grep 'UEFI OS' | tee /tmp/bbs3
+                    ;;
+                ncn-s*)
+                    efibootmgr | grep -iP '(pxe ipv?(4|6).*adapter)' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    efibootmgr | grep 'UEFI OS' | tee /tmp/bbs3
+                    ;;
+                ncn-w*)
+                    efibootmgr | grep -iP '(pxe ipv?(4|6).*adapter)' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    efibootmgr | grep 'UEFI OS' | tee /tmp/bbs3
+                    ;;
+                *)
+                    efi_fail_host
+                    ;;
+            esac
+            ;;
+        *Marvell*|HP|HPE)
+            # Removal file(s) ...
+            efibootmgr | grep -vi 'pxe ipv4' | grep -i adapter |tee /tmp/rbbs1
+            efibootmgr | grep -iP '(sata|nvme)' | tee /tmp/rbbs2
+            efibootmgr_prefix='0000,'
+            efi_trim
+            efi_specials
+            efi_remove
+            case $hostname in
+                ncn-m*)
+                    efibootmgr | grep -i 'port 1' | grep -i 'pxe ipv4' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    ;;
+                ncn-s*)
+                    efibootmgr | grep -i 'port 1' | grep -i 'pxe ipv4' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    ;;
+                ncn-w*)
+                    efibootmgr | grep -i 'port 1' | grep -i 'pxe ipv4' | tee /tmp/bbs1
+                    efibootmgr | grep -i cray | tee /tmp/bbs2
+                    ;;
+                *)
+                    efi_fail_host
+                    ;;
+            esac
+            ;;
+        *'Intel'*'Corporation'*)
+            # Removal file(s) ...
```

Comment:
```suggestion
            # Removal files ...
```

Reply:
It pains me but I'm going to omit amending the metal-lib.sh script with changes to keep the diff to the original script in 1.3/1.4 the same.